test/util/bspNet/waits.ts

```diff
@@ -112,10 +112,19 @@ export const waitForBspVolunteerWithoutSealing = async (
  *
  * @throws Will throw an error if the expected extrinsic or event is not found.
  */
-export const waitForBspStored = async (api: ApiPromise, checkQuantity?: number) => {
+export const waitForBspStored = async (api: ApiPromise, checkQuantity?: number, bspId?: string) => {
+  // TODO: add bsp ID to check if is not confirming storing
   // To allow time for local file transfer to complete (10s)
   const iterations = 100;
   const delay = 100;
+
+  if (bspId && checkQuantity && checkQuantity > 1) {
+    // We do this because `bspConfirmStoring` cannot happened in the same block in which a BSP submit a proof.
+    throw new Error(
+      "Cannot use `waitForBspStored` with a bspId and an amount of extrinsec bigger than 1."
```

Comment:
```suggestion
    // We do this because a BSP cannot call `bspConfirmStoring` in the same block in which it has to submit a proof, since it can only send one root-changing transaction per block and proof submission is prioritized.
    throw new Error(
      "Invalid parameters: `waitForBspStored` cannot be used with an amount of extrinsics to wait for bigger than 1 if a BSP ID was specified."
```